### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by Igetin**

### CODE REVIEW

File: src/translations/vnote_zh_CN.ts [hunk 1]
```diff
@@ -3431,7 +3431,7 @@
     </message>
     <message>
         <location filename="../vmainwindow.cpp" line="1045"/>
-        <source>Custom some standard shortcuts</source>
+        <source>Customize some standard shortcuts</source>
```
Comment (tamlok) [inline src/translations/vnote_zh_CN.ts]: `.ts` file is generated by Qt according to other source files, so there is no need to update this. I will update it before a new release.

File: src/vmainwindow.cpp [hunk 2]
```diff
@@ -1065,7 +1065,7 @@ void VMainWindow::initFileMenu()
     fileMenu->addAction(openConfigAct);
 
     QAction *customShortcutAct = new QAction(tr("Custom Shortcuts"), this);
```
Comment (tamlok) [inline src/vmainwindow.cpp]: How about changing this `Custom` to `Customize`, too?

File: src/translations/vnote_zh_CN.ts (hunk 1 above)
Comment (Igetin) [reply src/translations/vnote_zh_CN.ts]: Alright, I have no real experience in Qt so I didn't know this.

File: src/vmainwindow.cpp (hunk 2 above)
Comment (Igetin) [reply src/vmainwindow.cpp]: That one is fine either way, but I guess `Customize` would be better for the sake of consistency.

Comment (tamlok) [reply src/vmainwindow.cpp]: @Igetin Would you please send an update to this PR? Thanks!

Comment (Igetin) [reply src/vmainwindow.cpp]: @tamlok Done.

**Pull request merged**